src/runtime/wasm.ts

```diff
@@ -526,6 +527,18 @@ export class WasmParticle extends Particle {
     }
     this.innerParticle = this.exports[fn]();
     this.container.register(this, this.innerParticle);
+    // TODO(sjmiles): probably too soon: we need to render at least once, but we may have handle
+    // work pending. @shans says: if the particle has readable handles, onHandleUpdate is guaranteed
+    // to be called, otherwise we need `renderOutput` manually. Need to optimize this across all
+    // particle bases.
+    setTimeout(() => this.renderOutput(), 100);
+  }
+
+  renderOutput() {
+    // TODO(sjmiles): not yet implemented in CPP
```

Comment:
To stub this in CPP you need a function in the "JS-to-WASM" API section in arcs.cc:

```
EMSCRIPTEN_KEEPALIVE
void renderOutput(Particle* particle) {
}
```

Reply:
My tack was to not touch the CPP until I/we actually implement `renderOutput`.